azure-dataplane-tests/src/main/java/com/azure/data/schemaregistry/implementation/AzureSchemaRegistryImplBuilder.java

```diff
@@ -216,7 +216,7 @@ public AzureSchemaRegistryImplBuilder retryPolicy(RetryPolicy retryPolicy) {
     @Generated
     public AzureSchemaRegistryImpl buildClient() {
         HttpPipeline localPipeline = (pipeline != null) ? pipeline : createHttpPipeline();
-        String localApiVersion = (apiVersion != null) ? apiVersion : "2022-10";
+        String localApiVersion = (apiVersion != null) ? apiVersion : "2023-07-01";
```

Comment:
Swagger has changed:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/schemaregistry/azure-data-schemaregistry/swagger/README.md#code-generation-settings